React/Nextjs Project Post Setup Steps.md

```diff
@@ -0,0 +1,679 @@
+## Next.js project post setup steps
+
+
+#### Project example
+
+[JS React Example project](https://github.com/infinum/JS-React-Example)
+
+
+### General setup
+
+Before jumping into sections below, read pages for folder structure, naming conventions and other to see how general structure and setup should be in the project.
+
+Do not skip sections when doing setup below since some configurations depend on previous ones.
+
+In each section there will be a noted version for core packages because at the time you are reading this some configurations might
+change by package authors especially if new version exists. This doesn't mean setup will not work and if you encounter changes
+please inform your team lead.
+
+This post setup was done for Next.js **v9.5.3**. 
+
+
+### Folder restructuring
+
+1. Delete `styles` folder in the project root. In the project remove all code that was used from that folder.
+1. Create `src` folder in the project root and move `pages` folder into `src`
+1. Create `tests` folder in the project root. In there create `e2e`, `integration` and `unit` folders.
+1. Create `bin` folder in the project root with `build.sh` file
+1. Create `.github` folder in the project root with `CODEOWNERS` file
+1. Update `README` file
+1. Add `src/utils` folder
+1. Add `components/ui` and `components/containers` folders in `src` folder
```

Comment:
@DarkoKukovec we could make https://github.com/infinum/JS-React-Example public and use `create-next-app` for bootstrapping projects.
https://nextjs.org/docs/api-reference/create-next-app#options
```
create-next-app --example https://github.com/infinum/JS-React-Example/tree/chakra
```